docs/CONTRIBUTING-te.md

```diff
@@ -0,0 +1,261 @@
+- పుస్తకం పూర్తి కానట్లయితే మరియు ఇంకా పని చేస్తూ ఉంటే, [క్రింద](#in_process) వివరించిన విధంగా "`in process`" సంజ్ఞామానాన్ని జోడించండి.
+- ఒక వనరును [*Internet Archive's Wayback Machine*](https://web.archive.org) (లేదా ఇలాంటివి) ఉపయోగించి పునరుద్ధరించబడితే, "`archived`" సంజ్ఞామానం జోడించండి , [క్రింద](#archived) వివరించినట్లు. ఉపయోగించడానికి ఉత్తమ సంస్కరణలు ఇటీవలివి మరియు పూర్తి చేయబడ్డాయి.
+- డౌన్‌లోడ్ ప్రారంభించబడటానికి ముందు ఇమెయిల్ చిరునామా లేదా ఖాతా సెటప్ అభ్యర్థించబడితే, కుండలీకరణాల్లో భాషకు తగిన గమనికలను జోడించండి, e.g.: `(email address *అభ్యర్థించారు*, అవసరం లేదు)`.
+
+
+### Formatting
+
+- అన్ని జాబితాలు `.md` files మాత్రమే. [Markdown](https://guides.github.com/features/mastering-markdown/) syntax నేర్చుకోవడానికి ప్రయత్నించండి. ఇది సులభం!
+- అన్ని జాబితాలు సూచికతో ప్రారంభమవుతాయి. అక్కడ అన్ని విభాగాలు మరియు ఉపవిభాగాలను జాబితా చేసి లింక్ చేయాలనే ఆలోచన ఉంది. అక్షర క్రమంలో ఉంచండి.
+- విభాగాలు స్థాయి 3 శీర్షికలను ఉపయోగిస్తున్నాయి (`###`),మరియు ఉపవిభాగాలు స్థాయి 4 శీర్షికలు (`####`).
+
+ఆలోచన ఏంటంటే:
+
+- `2` ఖాళీ పంక్తులు చివరి లింక్ మరియు కొత్త విభాగం మధ్యన.
+- `1` ఖాళీ పంక్తులు శీర్షిక మరియు దాని విభాగం యొక్క మొదటి లింక్ మధ్యన.
+- `0` ఖాళీ పంక్తులు రెండు లింకుల మధ్యన.
+- `1` ఖాళీ పంక్తులు ప్రతి చివర `.md` file మధ్యన.
+
+Example:
+
+```text
+[...]
+* [An Awesome Book](http://example.com/example.html)
+                                (blank line)
+                                (blank line)
+### Example
+                                (blank line)
+* [Another Awesome Book](http://example.com/book.html)
+* [Some Other Book](http://example.com/other.html)
+```
+
+- `]` మరియు `(`  మధ్య ఖాళీలు పెట్టవద్దు:
+
+    ```text
+    BAD : * [Another Awesome Book] (http://example.com/book.html)
+    GOOD: * [Another Awesome Book](http://example.com/book.html)
+    ```
+
+- మీరు రచయితను చేర్చినట్లయితే, ` - ` (ఒకే ఖాళీలతో చుట్టుముట్టబడిన డాష్) వాడు:
+
+    ```text
+    BAD : * [Another Awesome Book](http://example.com/book.html)- John Doe
+    GOOD: * [Another Awesome Book](http://example.com/book.html) - John Doe
+    ```
+
+- లింక్ మరియు దాని ఆకృతి మధ్య ఒకే ఖాళీని ఉంచండి:
+
+    ```text
+    BAD : * [A Very Awesome Book](https://example.org/book.pdf)(PDF)
+    GOOD: * [A Very Awesome Book](https://example.org/book.pdf) (PDF)
+    ```
+
+- ఆకృతికి ముందు రచయిత వస్తాడు:
+
+    ```text
+    BAD : * [A Very Awesome Book](https://example.org/book.pdf)- (PDF) Jane Roe
+    GOOD: * [A Very Awesome Book](https://example.org/book.pdf) - Jane Roe (PDF)
+    ```
+
+- బహుళ ఫార్మాట్‌లు:
+
+    ```text
+    BAD : * [Another Awesome Book](http://example.com/)- John Doe (HTML)
+    BAD : * [Another Awesome Book](https://downloads.example.org/book.html)- John Doe (download site)
+    GOOD: * [Another Awesome Book](http://example.com/) - John Doe (HTML) [(PDF, EPUB)](https://downloads.example.org/book.html)
+    ```
+
+- పాత పుస్తకాల శీర్షికలో ప్రచురణ సంవత్సరాన్ని చేర్చండి:
+
+    ```text
+    BAD : * [A Very Awesome Book](https://example.org/book.html) - Jane Roe - 1970
```

Comment:
BAD/GOOD ni telugu lo rayagalamaa

Reply:
We can do that, but the thing is i have took some other translations as reference, where translation was omitted completely in the code parts between ``` and ```.

So what should i do.

reference : https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING-sv.md